test/kubemark/cloud-provider-config.sh

```diff
@@ -17,5 +17,5 @@
 CLOUD_PROVIDER="${CLOUD_PROVIDER:-gce}"
 CONTAINER_REGISTRY="${CONTAINER_REGISTRY:-gcr.io}"
 PROJECT="${PROJECT:-}"
-FULL_REGISTRY="${CONTAINER_REGISTRY}/${PROJECT}"
+FULL_REGISTRY="${FULL_REGISTRY:-}"
```

Comment:
Could you maybe rename this variable to `KUBEMARK_IMAGE_REGISTRY` to be more explicit?

Reply:
done